package.json

```
@@ -34,7 +34,7 @@
     "gas-report": "env ENABLE_GAS_REPORT=true yarn test:sol",
     "make:uml": "npx sol2uml ./contracts -b UNSRegistry,MintingManager,ProxyReader -o ./assets/uns.svg -i ./contracts/mocks",
     "analyze": "slither . --hardhat-artifacts-directory ./.artifacts --sarif slither.sarif --config-file .slither.json",
-    "postinstall": "([ -d ./.git ] && yarn husky install)"
+    "postinstall": "([ -d ./.git ] && yarn husky install) || true"
```

```suggestion
    "postinstall": "[ ! -d ./.git ] || yarn husky install"
```

This will probably preserve the `husky install` status, didn't check if it works though :)